contents/blog/posthog-vs-hotjar.md

```diff
@@ -0,0 +1,538 @@
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Sync data between platforms</td>
+        </tr>
+        <tr>
+            <td><strong>Sentry</strong></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Ingest Sentry errors for analysis</td>
+        </tr>
+        <tr>
+            <td><strong>Segment</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Ingest events from Segment</td>
+        </tr>
+    </tbody>
+</table>
+</div>
+
+> **Want more?** Visit the [PostHog the app directory](/apps) for a full list of supported integrations.
+
+## Privacy, compliance, and security
+
+Regulatory compliance can be a critical need for many teams, especially if they operate in financial or healthcare industries. Regulations such as HIPPA and GDPR can require teams to store data in certain locations, or to protect data in certain ways. 
+
+<div className="overflow-x-auto -mx-5 px-5">
+<table className="w-full mt-4" style="min-width: 400px;">
+    <thead>
+        <tr>
+            <td className="w-3/12"></td>
+            <td><strong>Hotjar</strong></td>
+            <td><strong>PostHog</strong></td>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td><strong>GDPR compliant</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>EU hosting or data residency available</td>
+        </tr>
+        <tr>
+            <td><strong>HIPAA compliant</strong></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td className="text-center">Self-host only</td>
+            <td>Business Associate Agreement available</td>
+        </tr>
+        <tr>
+            <td><strong>Data anonymization</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Remove personally identifiable information</td>
+        </tr>
+        <tr>
+            <td><strong>Self-hosting option</strong></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Self-host and keep complete control of your data</td>
+        </tr>
+    </tbody>
+</table>
+</div>
+
+## Frequently asked questions
+
+- [How is PostHog different?](#how-is-posthog-different)
```

Comment:
We don't need to repeat many of these headings at the end of the article? We can cut this down to just the ones which are actually in the FAQ section.

Reply:
Not sure what you mean here about it being auto-populated or uneditable -- and the right hand side will pull in all headings regardless of them being repeated here.

I've committed in the suggested change so that we don't repeat the already-used headings needlessly at the end of the article and only show the questions which are actually asked in this section.

Reply:
Relevant commit for comparison: https://github.com/PostHog/posthog.com/pull/5931/commits/09ab842bd70338b98539235b9a5387bfceede748